linbox/ring/ntl/ntl-rr.h

```diff
@@ -280,16 +280,17 @@ namespace LinBox
 	class UnparametricRandIter<NTL::RR> {
 		typedef NTL::RR Element ;
 	protected:
-		integer _size,_seed;
+		integer _size;
+        uint64_t _seed;
         const NTL_RR& _ring;
 	public:
 
 		UnparametricRandIter<NTL::RR> (const NTL_RR & F,
 					       const integer& size = 0,
-					       const integer& seed = 0) :
+					       const uint64_t seed = 0) :
                 _size(size), _seed(seed), _ring(F)
 		{
-			if (_seed == integer(0)) _seed = int64_t(time(NULL));
+			if (_seed == 0) _seed = time(NULL);
```

Comment:
uint64_t cast

Reply:
the cast is silently done by the compiler